velox/functions/lib/string/StringImpl.h

```diff
@@ -288,6 +288,17 @@ FOLLY_ALWAYS_INLINE void sha512(TOutString& output, const TInString& input) {
       folly::ByteRange((const uint8_t*)input.data(), input.size()));
 }
 
+// Compute the HMAC-SHA1 Hash.
+template <typename TOutString, typename TInString>
+FOLLY_ALWAYS_INLINE void
+HmacSha1(TOutString& output, const TInString& key, const TInString& data) {
```

Comment:
Nit : Please rename this function and the one below as hmacSha1 and hmacSha256 to be consistent with the function naming.

Reply:
done